src/sorcha/ephemeris/simulation_parsing.py

```diff
@@ -21,31 +21,40 @@ def mjd_tai_to_epoch(mjd_tai):
     return epoch
 
 
-def parse_orbit_row(row, epochMJD_TDB, ephem, sun_dict, gm_sun, gm_total):
+def parse_orbit_row(row, epochJD_TDB, ephem, sun_dict, gm_sun, gm_total):
     orbit_format = row["FORMAT"]
 
     if orbit_format not in ["CART", "BCART"]:
         if orbit_format == "COM":
+            # JD to MJD conversion if necessary
+            t_p = row["t_p"]
+            if t_p < 2400000.5:
+                t_p += 2400000.5
+
             ecx, ecy, ecz, dx, dy, dz = universal_cartesian(
                 gm_sun,
                 row["q"],
                 row["e"],
                 row["inc"] * np.pi / 180.0,
                 row["node"] * np.pi / 180.0,
                 row["argPeri"] * np.pi / 180.0,
-                row["t_p"],
-                epochMJD_TDB,
+                t_p,
+                epochJD_TDB,
             )
         elif orbit_format == "BCOM":
+            # JD to MJD conversion if necessary
+            t_p = row["t_p"]
+            if t_p < 2400000.5:
```

Comment:
Can we do this when we do the epoch conversion with the [one for epoch ](https://github.com/dirac-institute/sorcha/blob/846b251eb4412800dcb2c00bf2b63362e296997b/src/sorcha/ephemeris/simulation_setup.py#L93C8-L95C1)and change t_p here/ in the structure so it is tp_JD_TDB so its crystal clear what the units are.  
Good catch on the time labels being incorrect here.

Reply:
Actually on second thought it's fine to do it the conversion here, but let's be super clear about labeling variables.  t_p variable should have that is t_p_JD_TDB so it's crystal clear what unit is is in. Note there's ticket/issue in to rename t_p for the orbit inputs so it's explicitly t_p_MD_TDB 

We're requiring that all orbits be input in MJD for epoch and T0 do we need to do this test?  of the t_p < 2400000.5 and instead just to the conversion. If we're really worried that this the code could  give a warning in the log file if tp > 2400000.5: add a message to the log file saying the user should check their inputs to ensure the inputs are in MJD.


Question for @matthewholman  do we need to do the test the epoch and adjust in https://github.com/dirac-institute/sorcha/blob/main/src/sorcha/ephemeris/simulation_setup.py#L92-#L94  when wer're explicitly sending in MJD? I think in

Reply:
yeah - we can just assume that the user will supply a mjd and do the conversion anyway. I like the idea of adding something to the log file if it looks like something is in JD.